src/core/withExperiment.js

```diff
@@ -59,43 +59,18 @@ export const withExperiment = ({
     constructor(props: withExperimentInternalProps) {
       super(props);
 
-      log.info('[WithExperiment.constructor] props:', {
-        variantA: props.variantA,
-        variantB: props.variantB,
-      });
-
-      log.info('[WithExperiment.constructor] this.props:', {
-        variantA: this.props.variantA,
-        variantB: this.props.variantB,
-      });
-
       if (!this.isExperimentEnabled()) {
-        log.debug(`Experiment "${props.id}" is not enabled by config.`);
+        log.debug(`Experiment "${defaultId}" is not enabled by config.`);
```

Comment:
I believe this should be `props.id`

Reply:
oops! I didn't mean to change this.. thanks!